task/src/main/java/com/flipkart/flux/impl/task/AkkaTask.java

```diff
@@ -104,6 +108,12 @@ public void onReceive(Object message) throws Exception {
                 if (task != null) {
                     try {
                         // update the Flux runtime with status of the Task as running
+                        metricsClient.decCounter(new StringBuilder().
```

Comment:
Please move this to line 100 (after TaskAndEvents taskAndEvent = (TaskAndEvents) message;). Just after receiving the message the counter should be decremented.